src/service/config-store.ts

```diff
@@ -0,0 +1,56 @@
+import { HelpConfig } from '../help-config';
+import { ADMIN_CONFIG as adminConfig } from './configs/admin';
+import { AWARD_MANAGEMENT_CONFIG } from './configs/award';
+import { BB_HELP_CONFIG as bbHelpConfig } from './configs/bbhelp';
+import { config as bulkimportConfig } from './configs/bulkimport';
+import { config as churchConfig } from './configs/church';
+import { config as eduConfig } from './configs/edu';
+import { config as faithConfig } from './configs/faith';
+import { config as feConfig } from './configs/fe';
+import { config as fndliteConfig } from './configs/fndlite';
+import { config as gmkConfig } from './configs/grantmaking';
+import { config as marketplaceConfig } from './configs/marketplace';
+import { config as rexConfig } from './configs/rex';
+import { config as smallGroupsConfig } from './configs/smallgroups';
+import { STEWARDSHIP_MANAGEMENT_CONFIG } from './configs/stewardship';
+import { config as tcsConfig } from './configs/tcs';
+
+export type VALID_CONFIG_ID = 'admin'
+  | 'bbcrm'
+  | 'bbhelp'
+  | 'award-management'
+  | 'stewardship-management'
+  | 'chrch'
+  | 'edu'
+  | 'faith'
+  | 'fenxt'
+  | 'fenxt(fenxt.cdev)'
+  | 'fenxt(fenxt.ate)'
+  | 'fndlite'
+  | 'marketplace'
+  | 'renxt'
+  | 'smallgroups'
+  | 'tcs'
+  | 'gmk';
+
+export const CONFIG_STORE: { [k in VALID_CONFIG_ID]: HelpConfig } = {
+  admin: adminConfig,
+  'award-management': AWARD_MANAGEMENT_CONFIG,
+  bbcrm: bulkimportConfig,
+  bbhelp: bbHelpConfig,
+  chrch: churchConfig,
+  edu: eduConfig,
+  faith: faithConfig,
+  fenxt: feConfig,
```

Comment:
FE NXT has a couple locale folders, but they aren't working. It only has `en-au` and `en-ca`, but I can't get either those to open through this PR.

Reply:
interesting. it looks like i somehow copy-pasted wrong the FE config from [bb-help](https://dev.azure.com/blackbaud/Products/_git/skyux-spa-bb-help?path=%2Fsrc%2Fhelp_configs%2Ffe.ts&version=GBmaster&_a=blame) and left out the `customLocales`. will fix.